tests/rt.conf

```diff
@@ -120,6 +120,12 @@ RUN     | fv3_ccpp_gfs_v15p2
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | hera.intel     | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | orion.intel    | fv3         |
 RUN     | fv3_ccpp_gfs_v16beta                                                                                                           | standard    | cheyenne.intel | fv3         |
+RUN     | fv3_ccpp_gfs_v16beta_restart                                                                                                   | standard    | hera.intel     |             | fv3_ccpp_gfs_v16beta
```

Comment:
Why don't we run fv3_ccpp_gfs_v16beta* tests on wcoss?

Reply:
No idea, we never did.

Reply:
These tests are now run on both wcoss platforms.